06-Plotting-Time-Series-Rasters-in-R.Rmd

```diff
@@ -229,15 +241,34 @@ we adjust the layout to be a matrix of 4 columns and 4 rows.
 ```{r adjust-layout }
 #use level plot to create a nice plot with one legend and a 4x4 layout.
 levelplot(NDVI_stack,
-          layout=c(5, 3), #create a 4x4 layout for the data
+          layout=c(5, 3), #create a 5x3 layout for the data
           col.regions=cols, #add a color ramp
-          main="Landsat NDVI - Julian Days \nHarvard Forest 2014",
+          main="Landsat NDVI - Julian Days \nHarvard Forest 2011",
           names.attr=rasterNames)
 ```
 
-CHALLENGE -- change the X in each layer name to `Julian_` to label each plot.
-{: .notice }
+#Challenge: Divergent Color Ramps 
+When we used `gsub` to modify the tile labels we just used Day.  However, it
+would be nice to be more descriptive and use the full "JulianDay_" so that it is
+clear what is being denoted.  
 
+Create a plot with this change and use a divergent brown to green color ramp to
+represent the data.  Does having a divergent color ramp respresent the data
+better than a sequential color ramp (like "YlGn")?  Can you think of other data
+sets where a divergent color ramp may be best? 
 
+```{r challenge-code-levelplot-divergent, echo=FALSE }
```

Comment:
This is an odd challenge as it introduces new stuff not in the lesson which is OK IF we explain the new stuff some first. it seems like you want to get into cartography and color theory which could be a lesson on it's own. it might be worth considering this a bit further  - maybe via an issue?